lib/Common/Memory/SmallFinalizableHeapBucket.h

```diff
@@ -174,7 +173,7 @@ class HeapBucketGroup
         typedef typename SmallHeapBlockType<objectAttributes, TBlockAttributes>::BucketType BucketType;
         static BucketType& GetBucket(HeapBucketGroup<TBlockAttributes> * heapBucketGroup)
         {
-            CompileAssert(objectAttributes & FinalizeBit);
+            Assert(objectAttributes & FinalizeBit);
```

Comment:
I see, thanks!

---
In reply to: [89909533](https://github.com/Microsoft/ChakraCore/pull/2105#discussion_r89909533) [](ancestors = 89909533,89901040)